test/completion.js

```diff
@@ -29,28 +29,26 @@ describe('Completion', () => {
   });
 
   describe('Test completion STDOUT output', () => {
-    it('Returns the completion candidates for both options and installed generators', done => {
+    it.only('Returns the completion candidates for both options and installed generators', () => {
       const yocomplete = path.join(__dirname, '../lib/completion/index.js');
       const yo = path.join(__dirname, '../lib/cli');
 
       let cmd = 'export cmd="yo" && YO_TEST=true DEBUG="tabtab*" COMP_POINT="4" COMP_LINE="$cmd" COMP_CWORD="$cmd"';
       cmd += `node ${yocomplete} completion -- ${yo} $cmd`;
 
-      execFile('bash', ['-c', cmd], (err, out) => {
-        if (err) {
-          done(err);
-          return;
+      exec(cmd, (error, out, stderr) => {
+        console.log('stdout: ', out);
+        console.log('stderr: ', stderr);
```

Comment:
Testing and logging stderr here indeed helps a ton, you might get debugging statement this way.